<launch>
    <arg name="config_file" default="$(find strands_pedestrian_tracking)/config/config_Asus.inp" />
    <arg name="queue_size" default="10" />
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="ground_plane" default="/ground_plane" />
    <arg name="ground_hog" default="/groundHOG/detections" />
    <arg name="upper_body_detections" default="/upper_body_detector/detections" />
    <arg name="visual_odometry" default="/visual_odometry/motion_matrix" />
    <arg name="pedestrian_array" default="/pedestrian_tracking/pedestrian_array" />
    <arg name="pedestrian_image" default="/pedestrian_tracking/image" />
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="/opt/strands/strands_catkin_ws/devel/env.sh" user="$(arg user)" default="true"/>


    <node pkg="strands_pedestrian_tracking" type="pedestrian_tracking" name="pedestrian_tracking" output="log">
        <param name="config_file" value="$(arg config_file)" type="string"/>
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="ground_plane" value="$(arg ground_plane)" type="string"/>
        <param name="ground_hog" value="$(arg ground_hog)" type="string"/>
        <param name="visual_odometry" value="$(arg visual_odometry)" type="string"/>
        <param name="upper_body_detections" value="$(arg upper_body_detections)" type="string"/>
        <param name="pedestrian_array" value="$(arg pedestrian_array)" type="string"/>
        <param name="pedestrian_image" value="$(arg pedestrian_image)" type="string"/>
    </node>

</launch> 
